package.json

```diff
@@ -15,7 +15,9 @@
     "registry": "https://artifacts.aue1d.saasure.com/artifactory/api/npm/npm-okta"
   },
   "scripts": {
-    "lint:report": "grunt jshint",
+    "lint": "eslint ./lib ./test",
```

Comment:
Do we need to explicitly call out these directories, or can we simply do "eslint ." and then ignore any modules in an .eslintignore file?